### ISSUE TITLE
`gmp_rational` generates wrong `get_num` call

### ISSUE THREAD

**Issue by 0152la** [PollyLabs/library-metamorphic-testing#3]
Example `g++` error output:
```
test.cpp:286:104: error: ‘class __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __gmp_binary_expr<__gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_expr<__mpq_struct [1], __mpq_struct [1]>, __gmp_binary_minus> >, __gmp_binary_multiplies> >, __gmp_binary_multiplies> >, __gmp_binary_multiplies> >’ has no member named ‘get_num’; did you mean ‘get_val1’?
 q_class_w(operator-(operator*(r_19, operator*(r_19, operator*(r_19, operator-(r_19, r_19)))).get_num(), r_19.get_num()), r_19.get_den());
```

[get_num_test.zip](https://github.com/PollyLabs/library-metamorphic-testing/files/2961502/get_num_test.zip)

**Comment by diorga** [comment]
Should be fixed in the updated "gmp_rational" branch. I will make a pull request

**Comment by 0152la** [comment]
Fixed via pull request #4.
